### ISSUE TITLE
Druid---- value failed: [sum] already defined Error class:com.fasterxml.jackson.databind.JsonMappingException

### ISSUE THREAD

**Issue by xqwbx163** [metabase/metabase#8603]
### Database 
Druid
### Bugs
my database is Druid, 
![jdp_20180928161943](https://user-images.githubusercontent.com/20455410/46196702-6122f300-c33a-11e8-9da3-a3e7f0c1f8d8.png)

Select the summation operation through the interface,
select the first column to get the result, and the result column name is "sum".
 when i select the second column, show error:

```
Here's the full error message
java.lang.Exception: Unknown exception Instantiation of [simple type, class io.druid.query.groupby.GroupByQuery] value failed: [sum] already defined Error class:com.fasterxml.jackson.databind.JsonMappingException
```

the sum column name is repeated, why can't rename it like sum2 sum3, etc., it is not convenient to use.
  last use the SQL statement to achieve the result ,
 how to slove  this problem ? 
thank you

**Comment by xqwbx163** [comment]
i solve the problem ,rename the sum name then display more sum results. But display two sum column,you  can not division  like sum,sum2/100, the result only display sum2/100. the sum column not show.
who can help me ?

**Comment by camsaul** [comment]
Duplicate of #4166

**Comment by camsaul** [comment]
Already fixed. Fixed by #8987
